### PR TITLE
Childpage

### PR DESCRIPTION
**Pull request by meloneminze**
I build the css part of my first page.

### CODE REVIEW

File: client/src/components/Polaroid.js [hunk 1]
```diff
@@ -4,49 +4,48 @@ import defaultPic from '../assets/user-regular.svg';
...
   background-color: ${(props) => props.theme.colors.light};
   background-repeat: no-repeat;
   background-position: center;
-  background-size: 10rem;
-  text-align: center;
-  max-width: 100%;
-  height: 300px;
-  margin: 1rem;
-  flex-grow: 1;
+  background-size: 5rem;
```
Comment (Caruki) [inline client/src/components/Polaroid.js:31]: what do you want to accomplish with this property? if you want to set the size of the image you can use width and height. -> https://developer.mozilla.org/en-US/docs/Web/CSS/background-size

File: client/src/components/AddNewDataButton.js [hunk 2]
```diff
@@ -8,10 +8,12 @@ const AddNewDataButton = styled.button`
   border-color: ${(props) => props.theme.colors.light};
   border-radius: 5rem;
   border-width: 0.2rem;
+  height: 3rem;
+  margin: 0.6rem 0;
   box-shadow: 0rem 0.2rem 0.4rem rgba(172, 109, 26, 56);
   cursor: pointer;
   outline: none;
-  font: Roboto 1.5rem bold;
```
[the remark this answers is not in the document]
Comment (meloneminze) [reply client/src/components/AddNewDataButton.js:14]: Thank you!

File: client/src/components/AppHeader.js [hunk 3]
```diff
@@ -12,7 +12,9 @@ const Header = styled.header`
 const HeaderTitle = styled.h1`
   color: ${(props) => props.theme.colors.textPrimary};
   text-align: center;
-  padding: 10px;
+  padding: 0.3rem;
```
[the remark this answers is not in the document]
Comment (meloneminze) [reply client/src/components/AppHeader.js:15]: You are right!

File: client/src/components/MemberInfoInput.js [hunk 4]
```diff
@@ -30,30 +29,28 @@ background-color: ${(props) => props.theme.colors.input};
...
 `;
 
-const SimpleDataField = () => {
+const MemberInfoInput = () => {
   return (
     <TextareaContainer>
       <Headline
         placeholder="Adresse der Schule"
         minLength={1}
         maxLength={50}
```
[the remark this answers is not in the document]
Comment (meloneminze) [reply client/src/components/MemberInfoInput.js:43]: No, both ways working. I now use for all the strings.

File: client/src/components/Polaroid.js (hunk 1 above)
Comment (meloneminze) [reply client/src/components/Polaroid.js:31]: I want to resizing the background image like [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Resizing_background_images)

**Pull request merged**